R/across.R

```diff
@@ -634,9 +654,10 @@ expand_if_across <- function(quo) {
   call[[1]] <- quote(across)
   quos <- expand_across(quo_set_expr(quo, call))
 
-  # Select all rows if there are no inputs
+  # Select all rows if there are no inputs for if_all(),
+  # but select no rows if there are no inputs for if_any().
   if (!length(quos)) {
-    return(list(quo(TRUE)))
+    return(list(quo(empty)))
```

Comment:
We need `!!` to plug in the actual value. Do we need the `empty` variable at all?

```suggestion
    return(list(quo(!!(if_fn == "if_all"))))
```